JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/screens/dashboard/DashboardTopBar.kt

```diff
@@ -74,50 +79,55 @@ fun DashboardTopBar(
     Box(modifier = modifier) {
-        FocusGroup {
-            Row(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(top = 16.dp)
-                    .background(MaterialTheme.colorScheme.surface),
-                verticalAlignment = Alignment.CenterVertically
-            ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 16.dp)
+                .background(MaterialTheme.colorScheme.surface)
+                .focusRestorer(),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            key(ProfileScreenIndex) {
                 UserAvatar(
                     modifier = Modifier
-                        .restorableFocus()
-                        .focusRequester(focusRequesters[0]),
+                        .focusRequester(focusRequesters[0])
+                        .semantics {
+                            contentDescription =
+                                StringConstants.Composable.ContentDescription.UserAvatar
+                        },
                     selected = selectedTabIndex == ProfileScreenIndex,
                     onClick = {
                         onScreenSelection(Screens.Profile)
                     }
                 )
-                Row(
-                    verticalAlignment = Alignment.CenterVertically
-                ) {
-                    var isTabRowFocused by remember { mutableStateOf(false) }
+            }
+            Row(
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                var isTabRowFocused by remember { mutableStateOf(false) }
 
-                    Spacer(modifier = Modifier.width(20.dp))
-                    TabRow(
-                        modifier = Modifier
-                            .onFocusChanged {
-                                isTabRowFocused = it.isFocused || it.hasFocus
-                            },
-                        selectedTabIndex = selectedTabIndex,
-                        indicator = { tabPositions ->
-                            if (selectedTabIndex >= 0) {
-                                DashboardTopBarItemIndicator(
-                                    currentTabPosition = tabPositions[selectedTabIndex],
-                                    anyTabFocused = isTabRowFocused,
-                                    shape = JetStreamCardShape
-                                )
-                            }
+                Spacer(modifier = Modifier.width(20.dp))
+                TabRow(
+                    modifier = Modifier
+                        .onFocusChanged {
+                            isTabRowFocused = it.isFocused || it.hasFocus
                         },
-                        separator = { Spacer(modifier = Modifier) }
-                    ) {
-                        screens.forEachIndexed { index, screen ->
```

Comment:
I think we should add focusRestorer to TabRow as well.

Reply:
For the Top navigation bar, I have added the `focusRestorer` modifier to the parent `Row` container [here](https://github.com/android/tv-samples/blob/1153a37e5d06a55c41577db8c179b78aefafe038/JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/screens/dashboard/DashboardTopBar.kt#L87).